### ISSUE TITLE
Allow to disable tbkeys in calendar

### ISSUE THREAD

**Issue by strufepub** [wshanks/tbkeys#67]
I'm happy about tbkeys, as it helps me disable dangerous one hit disasters in Thunderbird (ignore thread, archive, etc).
I'm troubled, though, as it currently remains active when editing the calendar (create calendar entry and directly type its name), which frequently kills my calendar, or simply leads to omitted characters in the description of the entry.

Would it be able to include an option that allows the user to disable tbkeys for the calendar tab?

Thanks heaps!

**Comment by wshanks** [comment]
Yes, this is something I hadn't considered but having the keys active in the Calendar tab seems problematic (and maybe other tabs).
